README.md

```diff
@@ -4,22 +4,25 @@ FLORIS is a controls-focused wind farm simulation software incorporating
 steady-state engineering wake models into a performance-focused Python
 framework. It has been in active development at NREL since 2013 and the latest
 release is [FLORIS v3.0](https://github.com/NREL/floris/releases/tag/v3.0)
-in March 2022. The new ``v3`` version of the repository
-contains an architectural redesign of the software to enable improved
-performance in AEP calculation and controls optimization.
+in March 2022.
 
-We are actively seeking beta testers for the new framework. If you are interested
-in using FLORIS to conduct studies of a wind farm or extending FLORIS to include
-your own wake model, please get in touch! Join the conversations at
-[GitHub Discussions](https://github.com/NREL/floris/discussions/categories/v3-design-discussion).
+The software is in active development and engagement with the development team
+is highly encouraged. If you are interested in using FLORIS to conduct studies
+of a wind farm or extending FLORIS to include your own wake model, please join
+the conversation in [GitHub Discussions](https://github.com/NREL/floris/discussions/categories/v3-design-discussion)!
 
-For more context and background on previous work in FLORIS, see the
+This documentation is new since FLORIS v3, and it is a work in progress.
+For context and background on previous work in FLORIS, see the legacy
 documentation at http://floris.readthedocs.io/.
```

Comment:
To avoid confusion, should we before this link provide the link to the new github pages docs?

Reply:
Good point, I've updated this